Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -599,6 +599,7 @@ open class ChatMessageContentView: _View, ThemeProvider {
             textView?.dataDetectorTypes = .link
             textView?.isScrollEnabled = false
             textView?.backgroundColor = .clear
+            textView?.textContainer.lineBreakMode = .byTruncatingTail
```

Comment:
This will affect the message list as well. If we want to add the truncating in the message list too, we need a button to expand the message in the message list. 

At least for now, we can add this only in the Popup Actions view. Recently I've added a new function `ChatMessageActionsTransitionController.makeMessageContentView()` where we can change the message content view only for the popup actions view. The problem here is the `textView` is not created yet at this point :/ It will only be created once `createTextView()` is called.

So I can see two ways we can fix this:
1. Intoduce a new property `ChatMessageContentView.isPresentedInPopupActions: Bool` and set it to true in `ChatMessageActionsTransitionController.makeMessageContentView()` and then in `createTextView()` we only set the `.byTruncatingTail` if it is presented in popup actions.
2. Introduce closures or a factory of the different components in `ChatMessageContentView` so that we can easily replace the logic of `createTextView()` and the other views.

I actually prefer option 2, since it is more scalable and flexible, where 1. feels like a quick and dirty soluton.

Reply:
@nuno-vieira I don't see this behaviour 🤔

Reply:
@adamrushy Nevermind, I see, this is only truncated when the height is limited 👍